res/map/AWBWMapConverter.py

```diff
@@ -0,0 +1,211 @@
+
+import sys
+import math
+import re
+import csv
+
+
+def convertFile(infile,outfile):
+	reader = csv.reader(infile)
+	# Fun fact: this used to write out to the file once for every loop iteration. That was... not exactly what you'd call efficient. This is... more so.
+	outstring = ""
+	for line in reader:
+		for num in line:
+			try:
+				num = int(num)
+			# Teleporter tiles are a blank value in AWBW, so having -1 is a desired behavior for if/when we implement those.
+			except ValueError:
+				num = -1
+			outstring += indexToTerrainCode(num)
+		outstring += "\n"
+	outfile.write(outstring)
+	return
+
+def main(names):
+	for name in names:
+		try:
+			infile = open(name)
+			# Split the file extension off, and replace it with .map
+			outname = re.sub(r'\..*$','.map',name)
```

Comment:
I'd like to also replace any spaces in the filename with underscores. We already handle reading those on the Java side. I think you can accomplish this by just tacking `.replace(' ', '_')` to the end if this line.